.gitignore

```diff
@@ -1,8 +1,11 @@
 *~
 !.*
 
+.DS_Store
```

Comment:
IMHO this Mac-specific file type should rather go into your [`~/.gitignore_global`](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html) file

Reply:
I agree, but I also like having redundancy. Those things have a tendency to sneak into a repo when you're not looking. I'll remove it from here anyway.